doozer/doozerlib/cli/scan_osh.py

```diff
@@ -59,14 +50,23 @@ async def trigger_scan(self, nvrs: list):
                     self.runtime.logger.error("Invalid RHEL version")
                     return
 
-                cmd = cmd_template.format(f"rhel-{rhel_version}-x86_64", nvr)
+                cmd = cmd_template.format(config=f"rhel-{rhel_version}-x86_64", nvr=nvr)
+
+            message = f"Ran command: {cmd}"
 
-            self.runtime.logger.info(f"Running command: {cmd}")
+            if not self.dry_run:
+                fire_and_forget(self.runtime.cwd, cmd)
+            else:
+                message = "[DRY RUN] " + message
 
-            # Comment out the below line for testing.
-            fire_and_forget(self.runtime.cwd, cmd)
+            self.runtime.logger.info(message)
 
     async def run(self):
+        tags = self.runtime.get_errata_config()["brew_tag_product_version_mapping"].keys()
+        for tag in tags:
+            major, minor = self.runtime.group.replace("openshift-", "").split(".")
+            self.brew_tags.append(tag.replace("{MAJOR}", major).replace("{MINOR}", minor))
```

Comment:
`tag.format(MAJOR=major, MINOR=minor)` instead of .replace?

Reply:
ahh yes, apologies. fixed in https://github.com/openshift-eng/art-tools/pull/51/commits/c4c66665b3c5b9cc9d9b473fb0af3c5dbdf1253c